arduino-ide-extension/src/browser/style/settings-dialog.css

```diff
@@ -93,4 +93,5 @@
 
 .p-Widget.dialogOverlay .dialogBlock .dialogContent.additional-urls-dialog {
     display: block;
+    overflow: hidden;
```

Comment:
How is this change related to the primary button position?

Reply:
It's not related to the primary button position, working on this fix I noticed that there was an overflow problem above the buttons in the "Additional Boards Manager URLs" dialog:

![image](https://user-images.githubusercontent.com/94986937/190402318-8efcd8d6-c272-4715-b591-c8c09e1524f0.png)

Now it appears like this:

![image](https://user-images.githubusercontent.com/94986937/190404015-3f585df4-a5bf-44d2-ad01-5e9afb86e17d.png)

Reply:
Great. Thanks for the fix 💪